### PR TITLE
Add Product Sapling to planning

### PR DESCRIPTION
**Pull request by dnewh**
This adds a brief page about the Product Sapling to the planned features
section. This includes a couple of the preliminary wireframes.

Signed-off-by: Davey Newhall <newhall@bitwise.io>

### CODE REVIEW

File: community/planning/product_sapling.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!--
+  Copyright 2018-2021 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+# Product Sapling
+
+The [grid repository](https://github.com/hyperledger/grid/tree/master/ui)
+repository includes the UI for Grid functionality. This document shows some
```
Comment (RyanLassigBanks) [inline community/planning/product_sapling.md:10]: s/grid repository repository/grid repository

**Review by RyanLassigBanks — CHANGES_REQUESTED** [review]

File: community/planning/product_sapling.md (hunk 1 above)
Comment (dnewh) [reply community/planning/product_sapling.md]: updated

**Review by RyanLassigBanks — APPROVED** [review]

**Review by dplumb94 — APPROVED** [review]
Looks good

**Review by shannynalayna — APPROVED** [review]
Tested

**Pull request merged**